filebeat/crawler/prospector_log.go

```diff
@@ -31,7 +29,7 @@ func NewProspectorLog(p *Prospector) (*ProspectorLog, error) {
 	}
 
 	// Init File Stat list
-	prospectorer.harvesterStats = make(map[string]input.FileStat)
+	prospectorer.harvesterStates = make(map[string]input.FileState)
```

Comment:
already having some changes on this line how about: `prospectorer.harvesterStates = map[string]intput.FileState{}` ?

Is there a chance to get somewhat shorter names?